_posts/2025-05-19-announcement.md

```diff
@@ -1,8 +1,249 @@
+5. Once you have the dataset, you can fine-tune a model on it!
+
+See the full
+[execution](https://marin.community/data-browser/experiment?path=gs%3A%2F%2Fmarin-us-east1%2Fexperiments%2Fexp963_cascade_finemath-fa55e6.json)
+of the FineMath replication experiment.
+
+![Datashop]({{ site.baseurl }}/assets/images/posts/announcement-datashop-diagram.png)
+
+## Final remarks
+
+The future of AI should be open-source in the strongest sense: researchers and developers alike
+should be able to not just use AI, but to contribute directly back to it.
+We have built Marin, an open lab, that provides the infrastructure to allow this to happen.
+Of course training strong foundation models requires significant resources,
+and building communities is hard.
+Nonetheless, we must try.
+We have benefited greatly from an open Internet,
+open-source software, Wikipedia, and many assets in the public domain.
+At one point, these were also crazy ideas that were more likely to fail than to succeed.
+
+This is the beginning of the Marin journey.
+There is so much to do: we'd like to
+[try efficient linear attention architectures](https://github.com/marin-community/marin/issues/1313),
+[add long context support](https://github.com/marin-community/marin/issues/1314),
+support more languages, support multimodality, enhance reasoning with RL-based
+post-training, and you might have ideas that we haven't thought of.
+Come join us ([Discord](https://discord.gg/J9CTk7pqcM), [GitHub](https://github.com/marin-community/marin/))
+and let us deepen our scientific understanding and build the best models together!
+
+# Acknowledgements
+
+So many people and organizations have supported Marin, to whom we are greatly indebted:
+- First, we would like to thank Zak Stone and the [Google TPU Research Cloud
+(TRC)](https://sites.research.google/trc/) team.  Nearly all of the compute used
+for Marin comes from TRC.  This project might not have even gotten started
+without their support.
+- The Google JAX team, especially Roy Frostig, Sharad Vikram, Matthew Johnson,
+Yash Katariya, and Skye Wanderman-Milne, and Allen Wang from the TPU team helped
+us make good use of our TPU allocation.
+- The Anyscale team, especially Robert Nishihara and Richard Liaw, helped us
+with Ray, which Marin builds on.
+- This work originated from the Stanford Center for Research on Foundation
+Models (CRFM) and the Human-Centered AI Institute (HAI).  We are grateful for
+their support and fostering the belief that academia has an important role to
+play in the era of industrialized AI.
+- We would like to thank Matthew Ding, Kevin Klyman, Russell Power, and Cathy
+Zhou for their contributions to the project.
+- We would also like to thank the many people, including members of the Stanford
+AI Lab and Stanford NLP group, who have given advice and contributed valuable
+discussions.
```

Comment:
- We would also like to thank [Weights And Biases](https://wandb.ai/site/) for logging our many, many experiments.